Backend/src/resolverMap.ts

```diff
@@ -53,11 +53,11 @@ const resolverMap: IResolvers = {
   Mutation: {
     addUser: (_, { input }) => {
       const email: string = input.email;
-      if (!validateEmail(email)) {
-        throw new UserInputError("Email Argument invalid", {
-          invalidArgs: "email",
-        });
-      }
+      // if (!validateEmail(email)) {
+      //   throw new UserInputError("Email Argument invalid", {
+      //     invalidArgs: "email",
+      //   });
+      // }
```

Comment:
I don't know why but this check is so aggressive that it basically doesn't accept any emails

Reply:
Okay. Either we just completely remove it then, like you have here, or we need to find an alternative solution.

Reply:
Maybe we need another regular expression